pkg/network/networknamescheme/networknamescheme.go

```diff
@@ -19,4 +19,35 @@
 
 package networknamescheme
 
+import (
+	"fmt"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
 const DefaultNetworkName = "eth0"
+
+func GetNetworkNameSchemeMap(vmi *v1.VirtualMachineInstance) map[string]string {
```

Comment:
How about `CreateNetworkToLinkScheme`?
Also, pleas pass in only the information needed to create the data (i.e. `vmi.Spec.Networks`).

Reply:
the naming issue of networks/interfaces is indeed problematic but I don't think links should be into this.. we have already too many names for "network".